File: Blind_75_LeetCode_Questions/c++/Find_Minimum_in_Rotated_Sorted_Array.cpp
class Solution {
public:
    int findMin(vector<int>& nums) 
    {
        int head = 0; 
        int mid = 0;
        int tail = nums.size() - 1;
        while(head < tail)
        {
            mid = (head + tail) / 2;
            if(nums[mid] > nums[tail])
                head = mid + 1;
            else 
                tail = mid;
        }
        return nums[head];

    }
};

File: Blind_75_LeetCode_Questions/java/Array/Q153_Find_Minimum_in_Rotated_Sorted_Array/Find_Minimum_in_Rotated_Sorted_Array.java
class Solution {
    public int findMin(int[] nums) 
    {
        int len = nums.length;
        int val = nums[0];

        for(int i = 1; i < len; ++i)
        {
            if(nums[i] < val)
                val = nums[i];            
        }
        return val;
        
    }
}
